numpy/core/src/multiarray/dragon4.c

```diff
@@ -2424,14 +2450,14 @@ Dragon4_PrintFloat128(char *buffer, npy_uint32 bufferSize, FloatVal128 value,
     }
 
     /* deconstruct the floating point value */
-    floatExponent = GetExponent_F128(&value);
-    floatMantissa = GetMantissa_F128(&value);
```

Comment:
Might be convenient to add a
```
/* Create a mask with the lowest n bits set */
uint64_t mask_lowest_64(int32_t n) {
    return ~(~0 << n);
}
```
function, which saves from having to count the `F`s